proposal.rst

```diff
@@ -171,24 +188,27 @@ organisation(s)?**
 .. we don't have a problem with that, but it's helpful if we're aware of it
 .. so that we know how many backup choices we might need.
 
-FILLME
+No
 
-Your Project
-============
+Implement Bindings for Go
```

Comment:
We meant you to put the project title at the top of the document rather than here. It's not a huge problem though!